jenkins/run

```diff
@@ -63,9 +62,13 @@ fi
 # in the container.
 find . -name __pycache__ -exec rm -r {} \; > /dev/null 2>&1
 
-podman run -it --userns=keep-id --pull=always \
+podman run --rm -it \
+    --pull=newer \
+    --userns=keep-id \
+    --env-host \
+    --ulimit nofile=65536:65536 \
     --volume ${_tmp}:/tmp:z \
     --volume ${_var_tmp}:/var/tmp:z \
-    --volume $(pwd):/home/pbench:z -w /home/pbench \
-    --env-host --ulimit nofile=65536:65536 \
-    --rm ${IMAGE_REPO}/pbench-devel:${_branch_name} /bin/bash -c "${*}"
+    --volume $(pwd):/home/pbench:z \
+    -w /home/pbench \
+    ${IMAGE} /bin/bash -c "${*}"
```

Comment:
Looking at `main`, we've got a few options there that we don't have here which you might want to consider:
```
    --init \
    --security-opt label=disable \
    ${EXTRA_PODMAN_SWITCHES} \
```
Having the escape hatch of that last one has proved convenient....  And, including the first would presumably allow you to drop the `/bin/bash -c` thingie.  You can probably skip the middle one, since you don't mind the `:z`'s.  🙂

Reply:
Done.

Reply:
Ah, sec, something failed, checking ...

Reply:
Okay, its gonna work now ... ;)